### PR TITLE
doc: update some dynamic parameters from acrn-dm usage 

### PR DESCRIPTION
**Pull request by Kunhui-Li**
1. Cpu affinty was set by pCPU ID which can't be obtained explictly by user. Use lapic ID instead which can be easily read from `/proc/cpuinfo` as `apicid`. (please refer to PR #7040)
2. Since PR #7035 has removed "pm_notify_channel" and "pm_by_vuart" in the launch script generation logic, this patch also removes the related elements("poweroff_channel" and "allow_trigger_s5") in the documentation.

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Comment by gvancuts** [comment]
I have left a number of comments in PR #7040, please make sure you synchronise this one after updating #7040. Thanks!

File: doc/developer-guides/hld/hld-devicemodel.rst
Comment (dbkinder) [inline doc/developer-guides/hld/hld-devicemodel.rst:61]: More than one lapic_id can be specified here, so should we make this:
```suggestion
               [--cpu_affinity lapic_ids] [--lapic_pt] [--rtvm] [--windows]
```

File: doc/developer-guides/hld/hld-devicemodel.rst
Comment (dbkinder) [inline doc/developer-guides/hld/hld-devicemodel.rst:79]:
```suggestion
       --cpu_affinity: comma-separated list of Service VM vCPUs assigned to this VM. A Service VM vCPU is
```

File: doc/developer-guides/hld/hld-devicemodel.rst
Comment (dbkinder) [inline doc/developer-guides/hld/hld-devicemodel.rst:80]:
```suggestion
            identified by its lapic ID."
```

File: doc/reference/config-options-launch.rst
Comment (dbkinder) [inline doc/reference/config-options-launch.rst:83]:
```suggestion
  A comma-separated list of Service VM vCPUs assigned to this VM. A Service VM vCPU is identified
```

File: doc/reference/config-options-launch.rst
Comment (dbkinder) [inline doc/reference/config-options-launch.rst:84]:
```suggestion
  by its lapic ID.
```

File: doc/user-guides/acrn-dm-parameters.rst
Comment (dbkinder) [inline doc/user-guides/acrn-dm-parameters.rst:206]:
```suggestion
   comma-separated list of vCPUs assigned to this VM. A vCPU is identified by its lapic ID.
```

File: doc/user-guides/acrn-dm-parameters.rst
Comment (dbkinder) [inline doc/user-guides/acrn-dm-parameters.rst:207]: I think this needs more information.  What's the difference between apicid and lapic_id?

File: doc/user-guides/acrn-dm-parameters.rst
Comment (dbkinder) [inline doc/user-guides/acrn-dm-parameters.rst:205]:
```suggestion
``--cpu_affinity <list of lapic_ids>``
```

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/user-guides/acrn-dm-parameters.rst
Comment (dbkinder) [inline doc/user-guides/acrn-dm-parameters.rst:213]:
```suggestion
   to assign vCPUs with lapic_id 1 and 3 to this VM.
```

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: doc/developer-guides/hld/hld-devicemodel.rst
Comment (Kunhui-Li) [reply doc/developer-guides/hld/hld-devicemodel.rst]: Updated.

File: doc/developer-guides/hld/hld-devicemodel.rst
Comment (Kunhui-Li) [reply doc/developer-guides/hld/hld-devicemodel.rst]: Updated.

File: doc/developer-guides/hld/hld-devicemodel.rst
Comment (Kunhui-Li) [reply doc/developer-guides/hld/hld-devicemodel.rst]: Updated.

File: doc/reference/config-options-launch.rst
Comment (Kunhui-Li) [reply doc/reference/config-options-launch.rst]: Updated.

File: doc/reference/config-options-launch.rst
Comment (Kunhui-Li) [reply doc/reference/config-options-launch.rst]: Updated.

File: doc/user-guides/acrn-dm-parameters.rst
Comment (Kunhui-Li) [reply doc/user-guides/acrn-dm-parameters.rst]: Updated.

File: doc/user-guides/acrn-dm-parameters.rst
Comment (Kunhui-Li) [reply doc/user-guides/acrn-dm-parameters.rst]: Updated.

File: doc/user-guides/acrn-dm-parameters.rst
Comment (Kunhui-Li) [reply doc/user-guides/acrn-dm-parameters.rst]: Updated.

File: doc/user-guides/acrn-dm-parameters.rst
Comment (YuanyuanZhao609) [reply doc/user-guides/acrn-dm-parameters.rst:207]: lapic_id is short for 'local apic id', is consistent with SDM , and is more precise. 
'apicid' is the description in ``/proc/cpuinfo`` in linux OS.

Do you have some suggestion for more information?

Comment (YuanyuanZhao609) [reply doc/user-guides/acrn-dm-parameters.rst]: Updated， please check if it's easier to understand.

File: doc/user-guides/acrn-dm-parameters.rst
Comment (dbkinder) [inline doc/user-guides/acrn-dm-parameters.rst:207]:
```suggestion
   Interrupt Controller (LAPIC). The unique ID of the LAPIC (lapic_id) is used to identify vCPU.
```

File: doc/user-guides/acrn-dm-parameters.rst
Comment (dbkinder) [inline doc/user-guides/acrn-dm-parameters.rst:208]:
```suggestion
   The ``lapic_id`` for a vCPU can be found in the Service VM file ``/proc/cpuinfo``, 
```

File: doc/user-guides/acrn-dm-parameters.rst
Comment (dbkinder) [inline doc/user-guides/acrn-dm-parameters.rst:209]:
```suggestion
   identified as ``apicid``.
```

**Review by dbkinder — CHANGES_REQUESTED** [review]
a few wording tweaks and then LGTM.  Thanks.

File: doc/user-guides/acrn-dm-parameters.rst
Comment (YuanyuanZhao609) [reply doc/user-guides/acrn-dm-parameters.rst]: updated

File: doc/user-guides/acrn-dm-parameters.rst
Comment (YuanyuanZhao609) [reply doc/user-guides/acrn-dm-parameters.rst]: updated

File: doc/user-guides/acrn-dm-parameters.rst
Comment (YuanyuanZhao609) [reply doc/user-guides/acrn-dm-parameters.rst]: updated

**Review by dbkinder — APPROVED** [review]
LGTM, thanks!

**Comment by dbkinder** [comment]
@Kunhui-Li @NanlinXie Do we need to wait until #7040 is merged before merging this PR?

**Review by dbkinder — APPROVED** [review]

**Pull request merged**
